person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @name = name,
             @age = age,
             @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Well done with your Person class. As part of the requirements, you are supposed to create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. So, Kindy Implements the has-many relation in Person by creating a method to add rentals in the Person class.

Reply:
well resolved! ✔️